nd4j/samediff-import/samediff-import-onnx/src/main/kotlin/org/nd4j/samediff/frameworkimport/onnx/ir/OnnxIRGraph.kt

```diff
@@ -82,13 +82,6 @@ class OnnxIRGraph(graphDef: Onnx.GraphProto,opMappingRegistry: OpMappingRegistry
             nodeNames.add(node.nodeName())
         }
 
-        if(indexToNode.isNotEmpty()) {
```

Comment:
@treo this was removed because op definitions already happen automatically and was causing clashes.